stable/mongodb-replicaset/values.yaml

```diff
@@ -53,14 +53,31 @@ persistentVolume:
 # Annotations to be added to the service
 serviceAnnotations: {}
 
+tls:
+  # Enable or disable MongoDB TLS support
+  enabled: false
+  # Please exchange with your own TLS CA by generating it via:
+  # $ openssl genrsa -out ca.key 2048
+  # $ openssl req -x509 -new -nodes -key ca.key -days 10000 -out ca.crt -subj "/CN=mydomain.com"
+  # After that you can base64 encode it and paste it here:
+  # $ cat ca.key | base64 -w0
+  cacert: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUMrakNDQWVLZ0F3SUJBZ0lKQU5MWjhRcE83NW9CTUEwR0NTcUdTSWIzRFFFQkN3VUFNQkl4RURBT0JnTlYKQkFNTUIyMXZibWR2WkdJd0hoY05NVGN3T0RJd01URXlNRE0wV2hjTk5EVXdNVEExTVRFeU1ETTBXakFTTVJBdwpEZ1lEVlFRRERBZHRiMjVuYjJSaU1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBCjR1NFl4YUEzaXdBMXJ1T1NwTjNWU3V0ODZyUFpaQ243NEN6aU5UT3NMUzJhRmNXTmhTbHY4UzNFK2pwdGkzUW4KK2N0YWFHelBFbERoWGpjdTNNQTQzZkFGSCtWSVZmLzUwTERvSUVXRTdSK3JVdkhKTnB1ZEdTU2JmdUZKNzIrMQoxWU93NFJLSmFNRE5zS3hHWVN1VU1QdnliaHVMZUplQWZwWVVBNjN2djRKQ0ROYjFVRWlUUmhWaVMreE5XeDk1CmNVOFlaeGJKQkpOdDBNV0RvcXE3SEMwVUV4UW9MVTRtTFIxOW9wSEpOWml0NjhkVEE3akNnb3pzYkhieXNKN08KZVRQc1o0OVVzZ091UlZGUHdEejhZa0pyS3NtS1ZnVzZVUDUzMUhxYitPMlBuNmlmTWJzOUpNMXp4aFhWMUUxLwoycWROaGE3YjdPMUsySXpvcW5GWUd3SURBUUFCbzFNd1VUQWRCZ05WSFE0RUZnUVVlR0MxWGdTQ0wxRkk4VzZECjJFM1gxT3ZWSDVzd0h3WURWUjBqQkJnd0ZvQVVlR0MxWGdTQ0wxRkk4VzZEMkUzWDFPdlZINXN3RHdZRFZSMFQKQVFIL0JBVXdBd0VCL3pBTkJna3Foa2lHOXcwQkFRc0ZBQU9DQVFFQUR2bE44RGl5bTA3UTRRaFMwaEdhZUpJZgp5RWYvRzlWbWNjRVZjMXZJV0xoV1Y0emVPRW4yeHd6VTVPT3pDTGxiVyt0WjlLb1l6dU9LMC9wNzhHWFVIQkwyCkpxWGxvejlXTVNwc3pGaWRKTTJWbmJpUWZJM3hiaDFidHA1bEIrVm0wWkg0OUI4by9KbUcwQzBVZ25pTVFUZW0KSGZYb1Q5K3QwaDlHTzJsWjdTTHFkTDB2eDQyOTdMQnIrVlgrUHhDYklUUlowcXE4emRXWmZNTStIcWpXTzlIVwp6TldWMUJEbVRPd1AydEtZREt1OHFseCttZDMybzYzeldkZVNVWTZBU2VzUFkxNDBPdnN4RVQzRzFMejhYUXdBCnlsM2F3QWh5N3JxdEtHTGpwY0ovY0VlOHphcHY0R2hpRkNZY2Zla3JMSStsa2xjVVg2VmtKNUtzV0NuN2lRPT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
Is there a point to leaving this "default" cert and key on here? We could just leave it blank and let the user fill it in?

Reply:
The reason was to have at least one valid cert to just let it work out of the box. I can remove it but this means that the user has to generate a cert before using the chart.